app/models/miq_report_result.rb

```diff
@@ -35,6 +35,17 @@ class MiqReportResult < ApplicationRecord
     end
   end
 
+  # These two hooks of a fix for the following BZ:
```

Comment:
Maybe something like this: These two hooks prevent temporary chargeback aggregation data to be retained in each miq_report_results row found in the following BZ: